webapp/pom.xml

```diff
@@ -42,9 +42,9 @@
                <artifactId>servlet-api</artifactId>
             </exclusion>
             <exclusion>
-               <groupId>javax.servlet</groupId>
-               <artifactId>jsp-api</artifactId>
-            </exclusion>
+			    <groupId>javax.servlet.jsp</groupId>
+			    <artifactId>javax.servlet.jsp-api</artifactId>
```

Comment:
undesired spaces

Reply:
Please @mozzy11 may you point to the lines that have white space?